bootstrap/src/main/resources/application.conf

```
@@ -1,6 +1,22 @@
 dbDriver = "org.postgresql.Driver"
 dbUrl = "jdbc:postgresql://localhost:5432/random_coffee"
 
+bootstrap {
+
+  db {
+    driver = ${dbDriver}
+    url = ${dbUrl}?currentSchema=public
+    user = "postgres"
+    user = ${?AUTH_DB_USER}
+    password = "postgres"
+    password = ${?AUTH_DB_PASSWORD}
+  }
+
+  liquibase {
+    changelog-path = "liquibase/changelog/main-changelog-all.xml"
```

Probably it's better to rename `main-changelog-XXX` to `bootstrap-changelog-XXX` for consistency
```suggestion
    changelog-path = "liquibase/changelog/bootstrap-changelog-all.xml"
```

---

Yep, good point, thank you